README.md

```diff
@@ -1,6 +1,8 @@
 # Rust Space API Implementation
 
-[![Build Status](https://travis-ci.org/coredump-ch/spaceapi-server-rs.svg?branch=master)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
+[![Travis branch](https://img.shields.io/travis/coredump-ch/spaceapi-server-rs/master.svg)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
```

Comment:
That's just the title text. I simply copied the template from shields.io. There's no difference besides small details in design.